components/entitlementStrategies.ts

```diff
@@ -76,19 +77,28 @@ export class IOIStrategy extends EntitlementStrategy {
     override async get(userId: string) {
         // Note: Relies on the "legacyContractDownloader" flag.
         if (!userAuths.has(userId)) {
-            log(LogLevel.ERROR, `No data found for ${userId}.`)
+            log(LogLevel.ERROR, `No user data found for ${userId}.`)
             return []
         }
 
         const user = userAuths.get(userId)
 
-        const resp = await user?._useService<string[]>(
-            `https://${this._remoteService}.hitman.io/authentication/api/userchannel/ProfileService/GetPlatformEntitlements`,
-            false,
-            {
-                issuerId: this.issuerId,
-            },
-        )
+        let resp: AxiosResponse<string[]> = undefined
+
+        try {
+            resp = await user?._useService<string[]>(
+                `https://${this._remoteService}.hitman.io/authentication/api/userchannel/ProfileService/GetPlatformEntitlements`,
+                false,
+                {
+                    issuerId: this.issuerId,
+                },
+            )
+        } catch (AxiosError) {
```

Comment:
In JS, this is a variable name and not a type. In addition, the error may not itself be an instance of AxiosError, it needs to be explicitly checked.

Reply:
Added some checks on error type.

Reply:
Added some checks on error types